test/kubemark/start-kubemark.sh

```diff
@@ -425,10 +462,14 @@ done
 
 echo "DBG: TENANT_SERVER_KUBECONFIGS: ${TENANT_SERVER_KUBECONFIGS}"
 
-RESOURCE_SERVER_KUBECONFIG="/kubeconfig/rp.kubeconfig"
-echo "DBG: RESOURCE_SERVER_KUBECONFIG: ${RESOURCE_SERVER_KUBECONFIG}"
+for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
```

Comment:
Try to make start-hollow-nodes concurrent for multiple RPs

Reply:
tracked in issue
https://github.com/CentaurusInfra/arktos/issues/1036